.travis.yml

```diff
@@ -117,3 +117,73 @@ jobs:
         - make edgeimage ARCH="arm64v8"
         - make edgesiteimage ARCH="arm64v8"
         - make bluetoothdevice_image
+
+  - stage: Test on amd64 and build Debian Packages for amd64, aarch64 and armv7l
```

Comment:
Please follow the convention that implement the scripts under `/hack`.
`.travis.yml` is just an entrance to trigger things.